releasenotes/notes/fix-inverse-cancellation-c7f4debcde4a705a.yaml

```
@@ -1,3 +1,6 @@
 fixes:
   - |
-    The transpilation pass :class`.InverseCancellation` now runs inside of flow controlled blocks. Previously, it ignores the pairs of gates in classical blocks that can be cancelled. Refer to `#13437 <https://github.com/Qiskit/qiskit/issues/13437>` for more details.
+    The transpilation pass :class:`.InverseCancellation` now runs inside of flow
+    controlled blocks. Previously, it ignores the pairs of gates in classical blocks
+    that can be cancelled. Refer to `#13437 <https://github.com/Qiskit/qiskit/issues/13437>`__ 
```

Given that we are fixing the formatting, should we also make this past tense?

```suggestion
    controlled blocks. Previously, it ignored the pairs of gates in classical blocks
    that could be cancelled. Refer to `#13437 <https://github.com/Qiskit/qiskit/issues/13437>`__ 
```